### DISCUSSION TITLE
Link header in lowercase while fetching via code

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#25560] [Q&A] [answered]
Link for pagination in response headers, the header key name is specified as "Link" in documentation, we are getting the same while trying with Rested/Postman

But while making call from code, the header name is referred as "link" instead of "Link"

Is this expected? I think this is odd one!, kindly clarify, let me know if anyone need further information regarding the issue, I here by attach a sample request for which I got the response.[NOTE: WHILE MAKING CALL FROM CODE, THE ISSUE ARISES]

sample request URI:  https://mastodon.social/api/v1/statuses/110581010929144290/favourited_by?limit=1

request heaader: Authorization : xxxxxxxxxxxx

getting proepr response and link value but link key name is not in camelcase, instead it is completely lowercase

**Comment by andypiper** [comment, marked as answer]
I believe you are referring to the HTTP response values here? if that is the case, then HTTP/2 mandates that all header values are returned in lowercase as part of the RFC itself ([ref](https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2)).

**Reply by sudhagar-mr-17051** [reply]
Thanks for the response

**Comment by nightpool** [comment]
All HTTP header names are case insensitive, as per the spec http://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2 in HTTP/1.1, by convention, many were transmitted with Title-Case names, leading some to mistakenly believe that they were case sensitive 

When HTTP/2 was introduced, it mandated that all headers be transmitted as lowercase to resolve the ambiguity of this

**Reply by sudhagar-mr-17051** [reply]
Thanks for the clarification
